hw/ip/rv_core_ibex/doc/checklist.md

```diff
@@ -117,35 +117,40 @@ Documentation | [DV_PLAN_DRAFT_COMPLETED][]           | Waived      | Plan creat
 Documentation | [TESTPLAN_COMPLETED][]                | Done        |
 Testbench     | [TB_TOP_CREATED][]                    | Done        |
 Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| N/A         |
-Testbench     | [TB_ENV_CREATED][]                    | Done        |
-Testbench     | [RAL_MODEL_GEN_AUTOMATED][]           | N/A         |
+Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
+Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | N/A         |
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Not Started |
```

Comment:
Ibex is already in V1. Aligning this to the newest template adds a few categories like `CSR_CHECK_GEN_AUTOMATED`, `FPV_MAIN_ASSERTIONS_PROVEN` and `FPV_REGRESSION_SETUP`. 
We can probably set these three to `N/A` - since Ibex has no CSRs and no FPV testbench - but I wanted to double check.

Reply:
I agree, but would be good to get confirmation from @udinator and @rswarbrick

Reply:
Ibex is verified in a separate repo. We verify this wrapper at the chip level (i.e. close the coverage on this module). Our [dashboard](https://docs.opentitan.org/hw/#processor-cores) has a separate heading for Ibex, which is where the status (and Ibex's checklist) should be tracked. IMO having a checklist for this testbench-less module does not seem useful.

Reply:
Alternately, we treat this the same way as `rstmgr` and others verified at chip level.

Reply:
Shall we move on and merge this, and decide later on whether to remove the Ibex checklist or not?

Reply:
Yeah, please file a follow-up to continue the discussion there.

Reply:
Tracked here: #3808